AU2/database/model/Assassin.py

```diff
@@ -173,6 +195,9 @@ def pseudonyms_until(self, ts: dt.datetime = get_now_dt()) -> Iterator[str]:
         Yields:
             the next pseudonym of the assassin that is valid before datetime `ts`.
         """
+        if not ts:

```

Comment:
We're checking for None here, so it's better to be explicit so that devs inputting non-`datetime` data by mistake don't accidentally get `dt.now()`

```suggestion
        if ts is None:
```